components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -145,6 +145,7 @@ export class ClusterService implements IClusterServiceServer {
                     name: req.name,
                     url: req.url,
                     applicationCluster: this.config.installation,
+                    region: req.region,
```

Comment:
We should validate the `region` is not empty, server-side. If I start an older workspace, and perform registration, I'll have an older version of the pre-compiled cmd we use for registration, and I'll end up inserting empty region records.

Reply:
> Alternatively, we could consider this field optional, in which case a cluster would just never become part of the regional set.

I like that, doesn't make a ton of sense to provide regions in single cluster installations, for instance.

Reply:
> doesn't make a ton of sense to provide regions in single cluster installations

Unless you later want to be able to add a second region to an already existing installation. I'd personally make it required going forward, as that will give us better consistency on what we can expect when doing cluster discovery. Up to you.

Reply:
I was also thinking that rolling this change out would be easier if the new property was optional. We could deploy and then change the workspace cluster registration any time.